tests/main/res/config_launch_infinite.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dfl-config": {
+    "OutputDir": "results/launch_infinite",
```

Comment:
I didn't see where this new file was used?

Reply:
In tests/main we add a test which has it in input configuration